README.md

```diff
@@ -11,7 +11,7 @@
 
 ```
 # paper-pagination
-Polymer pagination element. Input record count, pagesize to get list of page chooser.
+Polymer pagination element. Input record count, pagesize to get list of page chooser. [Demo](https://dannyphung.github.io/paper-pagination/components/paper-pagination/demo/)
```

Comment:
the demo link is no longer working, please update

Reply:
Ok, I Will work on it.